pubspec.yaml

```diff
@@ -32,15 +32,18 @@ dependencies:
   flutter_summernote: ^1.0.0
   flutter_svg: ^0.23.0+1
   flutter_typeahead: ^3.1.3
-  get: ^4.1.4
-  get_it: ^7.1.3
+  get: ^4.6.1
+  get_it: ^7.2.0
   google_sign_in: ^5.0.4
   html: ^0.15.0
   http: ^0.13.3
   intl: ^0.17.0
   mockito: ^5.0.17
   oauth2_client: ^2.2.2
-  photo_view: ^0.13.0
+  # photo_view: ^0.13.0
```

Comment:
@manjotsidhu There are some bugs in the deployed version of this package but have been fixed on GitHub. So, I have temporarily updated this.